### PR TITLE
FIX Windows build

### PR DESCRIPTION
**Pull request by guparan**
Title says it all :wink:

### CODE REVIEW

**Comment by marques-bruno** [comment]
Can one of the admins verify this patch?

File: src/component/forcefield/MechanicalMatrixMapperMOR.h [hunk 1]
```diff
@@ -16,6 +16,7 @@
 ******************************************************************************/
 #ifndef MECHANICALMATRIXMAPPERMOR_H
 #define MECHANICALMATRIXMAPPERMOR_H
+#include <ModelOrderReduction/initModelOrderReduction.h>
```
Comment (VannesteFelix) [inline src/component/forcefield/MechanicalMatrixMapperMOR.h]: isn't it useless here ?

File: src/component/loader/MatrixLoader.h [hunk 2]
```diff
@@ -16,6 +16,7 @@
 ******************************************************************************/
 #ifndef SOFA_COMPONENT_LOADER_MATRIXLOADER_H
 #define SOFA_COMPONENT_LOADER_MATRIXLOADER_H
+#include <ModelOrderReduction/initModelOrderReduction.h>
```
Comment (VannesteFelix) [inline src/component/loader/MatrixLoader.h]: same

**Comment by VannesteFelix** [comment]
Thanks for the needed PR @guparan

**Comment by marques-bruno** [comment]
[ci-build]

File: src/component/loader/MatrixLoader.h (hunk 2 above)
Comment (guparan) [reply src/component/loader/MatrixLoader.h]: same :v:

File: src/component/forcefield/MechanicalMatrixMapperMOR.h (hunk 1 above)
Comment (guparan) [reply src/component/forcefield/MechanicalMatrixMapperMOR.h]: Possible, I added it everywhere just in case :small_airplane:

**Pull request merged**